modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -3104,6 +3104,10 @@ input ObservationPropertiesInput {
   """
   groupIndex: NonNegShort
 
+  """
+  Set the notes for  thhe observer
+  """
+  observerNotes: String
```

Comment:
I'm not sure if this should be `NonEmptyString`. Any opinions?

Reply:
I don't think it matters that much, but we made the subtitle `NonEmptyString` so I guess we could be more consistent with that.